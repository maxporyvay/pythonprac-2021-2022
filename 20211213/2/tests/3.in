[97, 84, 33, 54, -2, 53, 35, 58, 59, 19, 68, 56, 40, 46, 23, 26, 67, 77, 56, 1, -8, -7, 80, 53, 49, 3, 44, 86, 41, 18, 59, 99, 18, -7, 26, 58, 32, 5, -2, 92, 4, 92, 95, 0, 41, 95, 62, 15, 27, 15, 71, 27, 64, 26, 4, 33, 79, 100, 35, 21, 89, 79, 83]