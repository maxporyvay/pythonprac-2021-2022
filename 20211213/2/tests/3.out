[-8, -7, -7, -2, -2, 0, 1, 3, 4, 4, 5, 15, 15, 18, 18, 19, 21, 23, 26, 26, 26, 27, 27, 32, 33, 33, 35, 35, 40, 41, 41, 44, 46, 49, 53, 53, 54, 56, 56, 58, 58, 59, 59, 62, 64, 67, 68, 71, 77, 79, 79, 80, 83, 84, 86, 89, 92, 92, 95, 95, 97, 99, 100]
